scripts/apps/monitoring/directives/AggregateSettings.js

```diff
@@ -66,6 +66,16 @@ export function AggregateSettings(desks, workspaces, session, preferencesService
                 WizardHandler.wizard('aggregatesettings').next();
             };
 
+            scope.shouldHideStep = function(code) {
```

Comment:
Same here, all functions need to be documented using Angular's `dgeni` format. See [here](https://github.com/wingedfox/dgeni-alive).